src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -32,85 +32,94 @@
   </div>
 
   <div class="card-body">
+    {% set formOptions = {
+      'attr': {
+        'data-order-id': orderForViewing.id,
+        'data-is-delivered': orderForViewing.isDelivered,
+        'data-is-tax-included': orderForViewing.isTaxIncluded,
+        'data-discounts-amount': orderForViewing.prices.discountsAmountRaw,
+        'data-price-specification': priceSpecification|json_encode|replace("'", "&#39;")|raw
+      }
+    } %}
+    {{ form_start(cancelProductForm, formOptions) }}
     <table class="table" id="orderProductsTable" data-currency-precision="{{ orderCurrency.precision }}">
       <thead>
-        <tr>
-          <th>
-            <p>{{ 'Product'|trans({}, 'Admin.Global') }}</p>
-          </th>
-          <th></th>
-          <th>
-            <p class="mb-0">{{ 'Price per unit'|trans({}, 'Admin.Advparameters.Feature') }}</p>
-            <small class="text-muted">{{ orderForViewing.taxMethod }}</small>
-          </th>
-          <th>
-            <p>{{ 'Quantity'|trans({}, 'Admin.Global') }}</p>
-          </th>
-          <th>
-            <p>{{ 'Stock Location'|trans({}, 'Admin.Global') }}</p>
-          </th>
+      <tr>
+        <th>
+          <p>{{ 'Product'|trans({}, 'Admin.Global') }}</p>
+        </th>
+        <th></th>
+        <th>
+          <p class="mb-0">{{ 'Price per unit'|trans({}, 'Admin.Advparameters.Feature') }}</p>
+          <small class="text-muted">{{ orderForViewing.taxMethod }}</small>
+        </th>
+        <th>
+          <p>{{ 'Quantity'|trans({}, 'Admin.Global') }}</p>
+        </th>
+        <th>
+          <p>{{ 'Stock Location'|trans({}, 'Admin.Global') }}</p>
```

Comment:
```suggestion
          <p>{{ 'Stock location'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Reply:
Mmmm, careful I think this modif could be outdated or related to a wrong rebase
This part has been modified by the stock location PR https://github.com/PrestaShop/PrestaShop/pull/17135
And since the cancel product column has already been integrated in this template by the partial refund PR, I don't think you need to make any modification to this file

Reply:
I got your html via the rebase I did 👍

Reply:
Considering the number of modification, it can't be the same as from the rebase ^^ But it doesn't matter it's only indentation modifications However you should make the modification asked by Louise